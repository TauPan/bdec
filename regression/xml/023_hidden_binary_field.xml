<protocol>
    <sequence name="c">
        <field name="" length="0" />
        <field name="b" length="8" />
    </sequence>

</protocol>
