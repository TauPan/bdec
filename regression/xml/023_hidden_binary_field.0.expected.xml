<c><b>00000000</b></c>